src/sentry/web/frontend/debug/debug_weekly_report.py

```diff
@@ -18,8 +18,6 @@
 
 from .mail import MailPreviewView
 
-DEBUG_ISSUE_STATES = True
```

Comment:
@ceorourke lol this is a hardcoded value? so useless

Reply:
🤷‍♀️ I didn't put it there lol

Reply:
@ceorourke lol i'm aware